modules/living_spaces_event/src/Form/LivingSpaceEventInviteUsersForm.php

```diff
@@ -3,7 +3,6 @@
 namespace Drupal\living_spaces_event\Form;
 
 use Drupal\Core\Entity\Element\EntityAutocomplete;
-use Drupal\Core\Entity\EntityTypeManagerInterface;
```

Comment:
no need to remove it.

Reply:
It was used in the construct but it is not used anymore